extensions/amp-sidebar/0.1/storybook/Basic.amp.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// To do: how to add CSS
+//import '!style-loader!css-loader!./Basic-styles.css';
+import * as Preact from '../../../../src/preact';
+import {storiesOf} from '@storybook/preact';
+import {withA11y} from '@storybook/addon-a11y';
+import {withKnobs} from '@storybook/addon-knobs';
+import withAmp from '../../../../build-system/tasks/storybook/amp-env/decorator.js';
+
+// eslint-disable-next-line
+storiesOf('amp-sidebar', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withA11y)
+  .addDecorator(withAmp)
+  .addParameters({extensions: [{name: 'amp-sidebar', version: 0.1}]})
+  .add('Basic Sidebar', () => {
+    return (
+      <div>
+        <div id="main">
+          <h1>Main Body</h1>
+          <p>
+            Use the buttons below to open / toggle the amp-sidebar. The sidebar
+            will be displayed with various navigational elements.
+          </p>
```

Comment:
Usually examples should just be the component itself without any description. If we like we can use the storybook docs addon which allows for this style of inline examples. See https://github.com/storybookjs/storybook/tree/master/addons/docs#docspage
/cc @dvoytenko

Reply:
Thanks for the input @wassgha .  I am happy to remove the description or move the entire story into a storybook doc.

One thing to note is that it appears that some components do not work in isolation.  For amp-list here, at a minimum a button outside of the component is needed to toggle the sidebar into view.

Sounds like we should establish a precedent on how we want to handle potential descriptions and other elements outside of the component in a story.

Reply:
Awesome!  Descriptions moved to comments as we discussed in latest commit.